common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -596,7 +605,8 @@ class SplitModuleCourseTests(SplitModuleTest):
     Course CRUD operation tests
     '''
 
-    def test_get_courses(self):
+    @patch('xmodule.tabs.CourseTab.from_json', side_effect=mock_tab_from_json)
```

Comment:
It doesn't seem to work that way, probably because `mock_tab_from_json` is a real function, not a Mock.